### ISSUE TITLE
Keybord focus not visible on buttons on admin page

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7379]
Both in normal and high contrast mode.

**Comment by sigurdm** [comment]
In normal mode it is visibly - but extremely hard to notice.
Which of these buttons has focus?:
![image](https://github.com/dart-lang/pub-dev/assets/8613953/f4668b68-a868-455b-9f3b-911e4fd5a8c8)

(hint: it is the top one)

**Comment by isoos** [comment]
I think this is a limitation of the material design components we are using:
https://m2.material.io/components/buttons#outlined-button

It feels like migrating to v3 would be an improvement, but I didn't really see any specific example on the focus highlight:
https://m3.material.io/components/buttons/accessibility
